ol8/profiles/ospp.profile

```diff
@@ -0,0 +1,207 @@
+documentation_complete: true
+
+title: 'OSPP - Protection Profile for General Purpose Operating Systems'
```

Comment:
@iokomin I would change the title to `[DRAFT] OSPP - Protection Profile for General Purpose Operating Systems`, and then when OL is OSPP approved, remove `[DRAFT]`

Reply:
@redhatrises please find [DRAFT] added to profile title.